index.html

```diff
@@ -75,7 +75,7 @@ <h1 class="info-title">Comodidades</h1>
           <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn3"></div>
           <div class="pointer-list__oval pointer-list__oval--gray" id="divBtn4"></div>
         </div>
-        <button href="#" class="button button--medium button--secondary">Ver galeria</button>
+        <button href="#" class="button button--sm button--secondary">Ver galeria</button>
```

Comment:
href is not an attribute from button, please use either an `a` tag or `onClick` attribute.

Reply:
done